### DISCUSSION TITLE
Picker to show functions and variables inside a file

### DISCUSSION THREAD

**Discussion by ryanwcyin** [helix-editor/helix#7228] [Q&A] [answered]
As an Emacs user, I have found the keybindings in Helix to be quite convenient after using it for several days. However, I am wondering if there is a similar feature to HELM Semantic/Imenu in Helix. Specifically, I am looking for a feature that would display a list of all the classes, functions, and variables in a file and allow me to search them using fuzzy search.

**Comment by gabydd** [comment, marked as answer]
``<space>s`` uses the lsp to get symbols in the current buffer, as well you can do ``<space>S`` to have a symbol picker in the current workspace
